sql/fxa_derived/fxa_amplitude_export_v1/query.sql

```diff
@@ -0,0 +1,100 @@
+WITH
+  base_events AS(
+  SELECT
+    *
+  FROM
+    `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_auth_20*`
+  WHERE
+    _TABLE_SUFFIX = FORMAT_DATE('%g%m%d', '2020-04-20')
+    AND jsonPayload.fields.event_type IN ( 'fxa_activity - cert_signed',
+      'fxa_activity - access_token_checked',
+      'fxa_activity - access_token_created' )
+    AND jsonPayload.fields.user_id IS NOT NULL ),
+  grouped_by_user AS (
+  SELECT
+    -- to prevent weirdness from timestamp field, use provided
+    -- submission date parameter as timestamp
+    CAST('2020-04-20' AS DATETIME) AS submission_timestamp,
```

Comment:
I don't believe we care about the actual timestamp of the event, except for the day it occurs. Let me know if you can think of use cases otherwise.

Reply:
I'm a bit ignorant about the mechanics here, but if we use the same timestamp for all these rollup events then will we see a giant spike in event counts within amplitude at the same timestamp? I'm wondering if its worth using either mode_first or mode_last here so that the timestamp reflects the first or last event that we logged per user. I think I would lean towards mode_first. Would that cause you issues?

Reply:
So, it would cause a spike for that timestamp, but I don't think I've seen an Amplitude chart that is any more granular than day. We can do something like that, but I'd need to verify first that these `timestamp` field is guaranteed to be on that day.

Reply:
FxA does look at sub-day charts on occasion, sometimes to understand what happens when a new version of the server code rolls out. Of course, in this case they would still have to wait a day to see any changes, so maybe that's no longer a use case for these events. Not trying to make more work, unless I can think of a better reason not to do it or I hear differently from Alex I'm ok moving forward like this.

Reply:
It really shouldn't be a problem. But this is a weird event so to mark it as a specific timestamp that is distributed across the day for clients seems off as well, since that's not _actually_ correct. Really up to you all.

Reply:
Agreed its weird, the user would have to understand for these events that the timestamp means "user was active on this time and potentially after, but we don't know". I asked alex if he has a use case but I suspect we can move forward with a generic timestamp.